seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCode.java

```diff
@@ -38,7 +39,13 @@ public enum CommonErrorCode implements SeaTunnelErrorCode {
             "'<catalogName>' table '<tableName>' unsupported get catalog table with field data types '<fieldWithDataTypes>'"),
     GET_CATALOG_TABLES_WITH_UNSUPPORTED_TYPE_ERROR(
             "COMMON-21",
-            "'<catalogName>' tables unsupported get catalog table，the corresponding field types in the following tables are not supported: '<tableUnsupportedTypes>'");
+            "'<catalogName>' tables unsupported get catalog table，the corresponding field types in the following tables are not supported: '<tableUnsupportedTypes>'"),
+    FILE_NOT_EXISTED(
+            "COMMON-22",
+            "'<identifier>' '<operation>' file '<fileName>' failed, because it not existed."),
+    WRITE_SEATUNNEL_ROW_ERROR(
+            "COMMON-22",
+            "'<connector>' write SeaTunnelRow failed, the SeaTunnelRow value is <seaTunnelRow>.");
```

Comment:
`WRITE_SEATUNNEL_ROW_ERROR` may be coded with  `COMMON-23`.

Reply:
Yep, thanks for remind. I will change it before this PR ready.